### PR TITLE
[com_users] all views: Add missing tooltips in search

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Summary of Changes

Simple PR to add missing tootip to all views in the com_users search filters.
#### Testing Instructions

Code review.

or
1. Apply patch
2. Go to Users -> User Groups, check the tootip in the search field.
3. Go to Users -> Viewing Access Levels, check the tootip in the search field.
4. Go to Users -> User Notes, check the tootip in the search field.
5. Activate debug in global config
6. Go to Users -> Manage, click "Debug Permissions Report" and check the tootip in the search field.
7. Go to Users -> User Groups, click "Debug Permissions Report" and check the tootip in the search field.

### CODE REVIEW

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on c3e7536988a531d8ca2f7b7acf04e85a58d17645<br /><br />@brianteeman 
Is it ok for having in `COM_USERS_SEARCH_IN_NAME="Search in user name, username or email. Prefix with ID: to search for a user ID."`

`user name` and `username` ? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10130">issues.joomla.org/joomla-cms/10130</a>.</sub>

**Comment by brianteeman** [comment]
That doesnt seem correct - I will need to check it in context later - i am
on my phone

On 29 April 2016 at 13:07, Marc-Antoine Thevenet <notifications@github.com>
wrote:

> I have tested this item [image: :white_check_mark:] successfully on
> c3e7536
> <https://github.com/joomla/joomla-cms/commit/c3e7536988a531d8ca2f7b7acf04e85a58d17645>
>
> @brianteeman <https://github.com/brianteeman>
> Is it ok for having in COM_USERS_SEARCH_IN_NAME="Search in user name,
> username or email. Prefix with ID: to search for a user ID."
>
> user name and username ?
> ------------------------------
> This comment was created with the J!Tracker Application
> <https://github.com/joomla/jissues> at issues.joomla.org/joomla-cms/10130
> <https://issues.joomla.org/tracker/joomla-cms/10130>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/10130#issuecomment-215694669>
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by andrepereiradasilva** [comment]
for the record, i don't like it either, if you can suggest a better text it will be welcomed.

**Comment by brianteeman** [comment]
i will do that in a separate PR

On 29 April 2016 at 13:49, andrepereiradasilva <notifications@github.com>
wrote:

> for the record, i don't like it either, if you can suggest a better text
> it will be welcomed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/10130#issuecomment-215702113>
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by andrepereiradasilva** [comment]
ok, please test this one.

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 1]
```diff
@@ -257,10 +257,13 @@ COM_USERS_REVIEW_HEADING="Review Date"
 COM_USERS_SEARCH_ACCESS_LEVELS="Search Viewing Access Levels"
 COM_USERS_SEARCH_ASSETS="Search Assets"
 COM_USERS_SEARCH_GROUPS_LABEL="Search User Groups"
-COM_USERS_SEARCH_IN_GROUPS="Search in title"
-COM_USERS_SEARCH_IN_NAME="Search in name"
-COM_USERS_SEARCH_IN_NOTE_TITLE="Search in notes, subject or username."
-COM_USERS_SEARCH_TITLE_LEVELS="Search for Access Levels."
+COM_USERS_SEARCH_IN_GROUPS="Search in group title. Prefix with ID: to search for a group ID."
+COM_USERS_SEARCH_IN_NAME="Search in user name, username or email. Prefix with ID: to search for a user ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_users.ini]: Please change to
Search in name, username or email. Prefix with ID: to search for a user ID."

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 2]
```diff
@@ -257,10 +257,13 @@ COM_USERS_REVIEW_HEADING="Review Date"
...
+COM_USERS_SEARCH_IN_NOTE_TITLE="Search in note subject, user name or username. Prefix with ID: or UID: to search for a note ID or user ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_users.ini]: Please change to
Search in subject, name or username. Prefix with ID: or UID: to search for a note ID or user ID

**Comment by andrepereiradasilva** [comment]
done, and for "COM_USERS_SEARCH_IN_NOTE_TITLE" ?

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 3]
```diff
@@ -257,10 +257,13 @@ COM_USERS_REVIEW_HEADING="Review Date"
...
+COM_USERS_SEARCH_IN_LEVEL_NAME="Search in access level name. Prefix with ID: to search for a access level ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_users.ini]: Please change to
Search in level name. Prefix with ID: to search for an access level ID."

**Comment by andrepereiradasilva** [comment]
done, and done

**Comment by andrepereiradasilva** [comment]
... and done.

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 4]
```diff
@@ -257,10 +257,13 @@ COM_USERS_REVIEW_HEADING="Review Date"
...
+COM_USERS_SEARCH_IN_ASSETS="Search in asset name or title."
+COM_USERS_SEARCH_TITLE_LEVELS="Search Access Levels"
+COM_USERS_SEARCH_USER_NOTES="Search User Notes"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_users.ini]: When are the three above used?

Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.com_users.ini]: they are the search field labels -> accessibility

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on c3e7536988a531d8ca2f7b7acf04e85a58d17645<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10130">issues.joomla.org/joomla-cms/10130</a>.</sub>

**Comment by brianteeman** [comment]
At some point we will have to review the text on all these search strings - i will add it to the en-gb tasklist<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10130">issues.joomla.org/joomla-cms/10130</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10130">issues.joomla.org/joomla-cms/10130</a>.</sub>

**Pull request merged**